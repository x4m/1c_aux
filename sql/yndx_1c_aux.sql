select binrowver(x::integer) from generate_series(1,100,7) x;