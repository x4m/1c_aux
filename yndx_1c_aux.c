/*-------------------------------------------------------------------------
 *
 * yndx_1c_aux.c
 *		Nothing useful
 * Copyright (c) 2019, PostgreSQL Global Development Group
 *
 * IDENTIFICATION
 *	  contrib/yndx_1c_aux/yndx_1c_aux.c
 *
 *-------------------------------------------------------------------------
 */

#include "postgres.h"


PG_MODULE_MAGIC;