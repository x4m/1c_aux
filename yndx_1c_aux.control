comment = 'Auxilary functions to run 1C ERP system on Yandex.Cloud MDB clusters'
default_version = '1.0'
module_pathname = '$libdir/yndx_1c_aux'
relocatable = true

