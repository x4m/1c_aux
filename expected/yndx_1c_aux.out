select binrowver(x) from generate_series(1,100,7) x;